.gitignore

```diff
@@ -23,6 +23,7 @@
 **/doc/api/
 .dart_tool/
 .flutter-plugins
+.flutter-plugins-dependencies
```

Comment:
what does this do?

Reply:
Auto generated file to track dependencies from plugins